Formula/libint.rb

```diff
@@ -0,0 +1,20 @@
+class Libint < Formula
+  desc "Library for computing electron repulsion integrals efficiently"
+  homepage "https://github.com/evaleev/libint"
+  url "https://github.com/evaleev/libint/archive/v2.7.2.tar.gz"
+  sha256 "fd0466ce9eb6786b8c5bbe3d510e387ed44b198a163264dfd7e60b337e295fd9"
+  license "GPL-2.0-or-later"
```

Comment:
this needs to be `license all_of: ["GPL-2.0-or-later", "GPL-3.0-or-later"]`

Reply:
I was wrong with this comment, this actually should be LGPL3+ and GPL3+, just fixed it.